recipes/urlquote/meta.yaml

```diff
@@ -0,0 +1,83 @@
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    # - {{ compiler('c') }}
+    - rust
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - six
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - urlquote
+    - urlquote.quoting
+
+
+about:
+  home: https://github.com/blue-yonder/urlquote
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGPL.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: MIT
+  # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
+  license_family: MIT
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See http://conda.pydata.org/docs/building/meta-yaml.html#license-file
+  license_file: LICENSE
+  summary: 'Fast percent encoding / decoding for python.'
+
+  # The remaining entries in this section are optional, but recommended.
+  description: |
+    urlquote is not intended to be a replacement for urllib. urllib is part of the Python standard
+    library and should be your go-to choice to quote and unquote URLs. However, should quoting or
+    unquoting of URLs be known to be a performance bottleneck and you are fine with the encodings
+    provided, then by all means have fun using this library.
+  doc_url: https://github.com/blue-yonder/urlquote/blob/v1.1.1/README.md
+  dev_url: https://github.com/blue-yonder/urlquote
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
```

Comment:
Feel free to add me to help with maintenance.

Reply:
Thanks, your help is welcome!